### DISCUSSION TITLE
Is the HPND license okay?

### DISCUSSION THREAD

**Discussion by Takahiro-Funahashi** [python-pillow/Pillow#6393] [Q&A] [answered]
In the URL below, the HPND license has the description "This License has been voluntarily deprecated by its author." Is it okay to keep the HPND license for python-pillow?

[HPND License] (https://opensource.org/licenses/HPND)

**Comment by nulano** [comment, marked as answer]
This was last discussed in #5975, with the conclusion that the deprecation refers to new projects only, and it's okay to keep the license for Pillow.
